themes/digital.gov/layouts/_default/404.html

```diff
@@ -32,11 +41,8 @@ <h1>{{- .Title | markdownify -}} </h1>
               {{- .Content -}}
             </div>
 
-            {{- partial "core/page-data.html" (dict "page" . "page_type" "resource" "branch" ($.Scratch.Get "branch")) -}}
-
-          </div>
+            {{- partial "core/page-data.html" (dict "page" . "page_type" "404" "branch" ($.Scratch.Get "branch")) -}}
```

Comment:
Passed 404 as the `page_type`, but turns out this property isn't even used per comment https://github.com/GSA/digitalgov.gov/blob/fe3932d6c132e45d858054ab01470d398a6768f6/themes/digital.gov/layouts/partials/core/page-data.html#L8